roles/kafka-mesos/tasks/destroy.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Register running brockers
+  shell: ./kafka-mesos.sh status | grep -c "id:.[0-9]"
+  register: existing_brockers
+  args:
+    chdir: "{{ kafka_mesos_install_dir }}"
+  tags:
+    - kafka-mesos
+  failed_when: existing_brockers.stdout == -1
+
+- name: Stop all Kafka brockers
+  run_once: true
+  sudo: true
+  command: "./kafka-mesos.sh stop 0..{{ existing_brockers.stdout | int - 1 }} --api http://{{ kafka_scheduler_hostname }}:7000"
```

Comment:
1. What if there are 0 brockers?  "stop 0..-1"?  Don't you need "when" condition?
2. What if there is 1 brocker?  Is "0..0" a correct syntax?  Did you test it?